src/API/Oauth2Client.php

```diff
@@ -237,10 +237,11 @@ private function dontPersist($name) {
      * @return Boolean Whether it exchanged the code or not correctly
      */
     public function exchangeCode() {
-        if (!isset($_REQUEST['code'])) {
+        $code = $_GET['code'] ?: $_POST['code'];
```

Comment:
it should be `$code = isset($_GET['code']) ? $_GET['code'] : $_POST['code'];` 
or `$code = $_GET['code'] ?? $_POST['code'];`